rfd/0078-login-rules.md

```diff
@@ -0,0 +1,703 @@
+---
+authors: Nic Klaassen (nic@goteleport.com)
+state: draft
+---
+
+# RFD 78 - Login Rules
+
+## Required Approvers
+
+- @klizhentas
```

Comment:
Should we add Security approvers? Engineering? I assume Sasha is for product, but should we CC Xin as well?

Reply:
I listed Sasha here because he proposed these ideas and worked pretty closely on this. Is this a suggestion to add more people to the list? Is there a policy of who to include here? I don't know.

Reply:
I've been following https://github.com/gravitational/teleport/blob/master/rfd/0000-rfds.md#required-approvers. There isn't a fixed policy that I know of, I tend double-check with my manager/stakeholders and try to cover the 3 areas.